db/migrate/20150902073723_create_sciences.rb

```diff
@@ -8,6 +8,7 @@ def change
       t.float :factor
       t.time :duration
       t.string :condition
+      t.string :name
```

Comment:
Migrations nachträglich bearbeiten ist böse.
Besser eine Migration generieren, die die Änderung vornimmt.

Wie das geht, findet ihr in den Rails Guides. ;)